### PR TITLE
Add an events guide

### LINKED ISSUE #111: Improve documentation and best practices around events
### Description

In https://github.com/2i2c-org/infrastructure/issues/790 we identified a few documentation improvements that would have made it easier for the folks running the event to know what to expect.

### Implementation details

Here are a few ideas below:

- [ ] Provide a checklist about what people should do before an event
    - Run the notebooks
    - Click the links etc

- [ ] Environment generation
    - It was confusing to have to go to a separate repository to learn how to build a docker image
    - More context around the instructions would be helpful
    - Make it clear that people can re-use the [quay.io](http://quay.io) images for other JupyterHubs etc
    - Improve explanations around what "quay.io" is in general
- [ ] Tutorials for "what do you do once the hub is set up?"
    - E.g., step by step guide to sharing a notebook on a hub
- [ ] Recommendations for how communities can triage participants ahead of time
    - E.g., recommend that communities run a notebook with some basic functionality on *binder* so that they know what skills they're expected to learn before signing up
- [ ] How to define multiple user profiles for use on the hub
    - It wasn't obvious how to do this as an admin

### PR DESCRIPTION
**Pull request by choldgraf**
This PR does a few things:

- Adds a section to the docs for "usecase guides". I imagine this is a place where we can put information that is useful for particular use-cases like "Pangeo Style" workflows, "teaching", "events", etc.
- Adds a short "events guide" with a few tips for running events on a hub. Over time we can update this with more best-practices as we learn them.
- Re-organizes the pages slightly so that the left sidebar is less cluttered, and nests the `network` page under a `topics` folder rather than being a top-level page.
- Updates the environment information after feedback from PaleoHackWeek

If nobody objects, I'd like to get this merged in and see if it is useful for https://github.com/2i2c-org/infrastructure/issues/810 and we can iterate on this content further when debriefing after the OpenScapes hub

closes #111

### CODE REVIEW

File: admin/howto/environment.md [hunk 1]
```diff
@@ -5,7 +5,92 @@ configured environment with base libraries, user interfaces and
 languages installed. This allows them to start working immediately,
 without having to install packages themselves.
 
-## Default user environment
+
+(environment/custom)=
+## Customize your user environment
+
+Whie all hubs [come with a default environment](environment/default), it is possible to create a custom user environment for the hub.
```
Comment (GeorgianaElena) [inline admin/howto/environment.md:12]:
```suggestion
While all hubs [come with a default environment](environment/default), it is possible to create a custom user environment for the hub.
```

**Review by GeorgianaElena — APPROVED** [review]
LGTM :rocket:

File: admin/howto/environment.md [hunk 2]
```diff
@@ -5,7 +5,92 @@ configured environment with base libraries, user interfaces and
...
+just last the duration of your user session. They will get wiped out
+when your user server is stopped, to ensure that you always start from
+a 'clean slate' environment.
+
+The recommended way is to put `%pip install <list-of-packages>` or
+`%conda install <list-of-packages>` in the first cell of any notebook
+you distribute, so when run it'll install necessary packages. For R,
+you can use `install.packages("package-name")` as you normally would.
+
+```{warning}
```
Comment (sgibson91) [inline admin/howto/environment.md:72]: What's the difference between the `%` and `!` prefixes in the 2 `pip` examples? Do they cause the packages to be installed in different places, or is that solely the `--user` flag?

(This is probably more for my own understanding than a requirement for the docs?)

**Review by sgibson91 — APPROVED** [review]
LGTM!

File: admin/howto/environment.md [hunk 3]
```diff
@@ -5,7 +5,92 @@ configured environment with base libraries, user interfaces and
...
+:color: primary
+Go to user environment template
+```
+
+### Temporarily install packages for a session
+
+You can temporarily install packages in your environment that will
+just last the duration of your user session. They will get wiped out
+when your user server is stopped, to ensure that you always start from
+a 'clean slate' environment.
```
Comment (damianavila) [inline admin/howto/environment.md:65]:
```suggestion
a 'clean state' environment.
```

Maybe?

**Review by damianavila — APPROVED** [review]
Minor comment, but LGTM.
Let's iterate on this one!

File: admin/howto/environment.md (hunk 3 above)
Comment (sgibson91) [reply admin/howto/environment.md:65]: What Chris has written is correct: https://www.dictionary.com/browse/clean--slate BUT it's a good idea to reduce the use English idioms in our docs as much as possible

Comment (sgibson91) [reply admin/howto/environment.md:65]: Maybe "always start from the default environment"? Or "provided environment"?

**Comment by choldgraf** [comment]
OK I believe that I've addressed all of the PR comments, thanks everybody for the helpful feedback! And agreed w/ @damianavila we should keep iterating on this!

**Pull request merged**